lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -1754,6 +1754,100 @@ A few notes:
 -   If a target array cannot accommodate all values (i.e., the length of source array plus `i` exceeds the target array length), the method throws an error.
 -   If provided a [typed array][@stdlib/array/typed] which shares an [`ArrayBuffer`][@stdlib/array/buffer] with the target array, the method will intelligently copy the source range to the destination range.
 
+<a name="method-slice"></a>
+
+#### Complex64Array.prototype.slice( \[begin\[, end]] )
+
+Returns a new typed array containing the extracted elements.
+
+```javascript
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+
+var arr = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 ] );
+
+var slicearr = arr.slice();
+// returns <Complex64Array>
+
+var len = slicearr.length;
+// returns 4
+
+var z = slicearr.get( 0 );
+// returns <Complex64>
+
+var re = realf( z );
+// returns 1.0
+
+var im = imagf( z );
+// returns 2.0
+
+z = slicearr.get( len-1 );
```

Comment:
```suggestion
var out = arr.slice();
// returns <Complex64Array>

var len = out.length;
// returns 4

var z = out.get( 0 );
// returns <Complex64>

var re = realf( z );
// returns 1.0

var im = imagf( z );
// returns 2.0

z = out.get( len-1 );
```

By convention, we typically prefer simple and consistent variable names (e.g., `x`, `y`, `z`, `out`, `fcn`, `f`, `arr`, etc).